### ISSUE TITLE
Cannot install Sails Js :(

### ISSUE THREAD

**Issue by IslamBesto** [npm/npm#9091]
I got this ERROR :'( 

```
npm ERR! Windows_NT 6.3.9600
npm ERR! argv "node" "C:\\Users\\islam\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "-g" "install" "sails"
npm ERR! node v0.12.7
npm ERR! npm  v2.13.2

npm ERR! shasum check failed for C:\Users\islam\AppData\Local\Temp\npm-21728-a6571732\registry.npmjs.org\uri-js\-\uri-js-1.4.2.tgz
npm ERR! Expected: b5d67221edc6b4be269146ee34082876ed729bd0
npm ERR! Actual:   979c5945c76517f7766cff1ce5aea1ca0fbfa60b
npm ERR! From:     https://registry.npmjs.org/uri-js/-/uri-js-1.4.2.tgz
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Windows\System32\npm-debug.log
```

**Comment by sinkingshriek** [comment]
This seems related to [this](https://github.com/npm/npm/issues/2701#issuecomment-26624532). Someone correct me if it's fixed on node v.12+.

**Comment by IslamBesto** [comment]
I'm still getting the same error :-1:

**Comment by othiym23** [comment]
There's a few things that could be happening here. A quick thing you can do, @IslamBesto, is run `npm cache clean` and then try to run the install again and see if perhaps something got busted in your cache. It happens less frequently with newer versions of npm, but it is possible.

You might also be running into a connectivity issue. You're running a new enough version of npm that you can use `npm ping`. Give that a shot and paste the results here, especially if they contain a stack trace or something other than `"ok": true` in the payload that gets returned.

@sinkingshriek The old Node.js versioning scheme was odd _minor_ versions were testing builds, and even _minor_ versions were the release versions of Node. The issue you linked to was an issue in the built-in `crypto` module in versions of Node.js 0.11 prior to 0.11.12 (I think), and are definitely fixed in Node `0.12.x`. It could be a problem if the packages are question were _published_ with Node.js 0.11, but I was able to install `uri-js@1.4.2` locally with no issues, so that doesn't seem to be the issue here.

**Comment by ehsalazar** [comment]
We haven’t heard back and we’re trying to clean up some old issues. If this is still a problem, can you please reply and let us know? We’ll be happy to reopen if necessary.
